resources/functionMap_php82delta.php

```diff
@@ -0,0 +1,29 @@
+<?php // phpcs:ignoreFile
+
+/**
+ * Copied over from https://github.com/phan/phan/blob/8866d6b98be94b37996390da226e8c4befea29aa/src/Phan/Language/Internal/FunctionSignatureMap_php80_delta.php
+ * Copyright (c) 2015 Rasmus Lerdorf
+ * Copyright (c) 2015 Andrew Morrison
+ */
+
+/**
+ * This contains the information needed to convert the function signatures for php 8.0 to php 7.4 (and vice versa)
```

Comment:
I think this should be 8.2 right?

And is this the difference between 7.4 and 8.2, or between 8.1 and 8.2?

Reply:
Yeah, see PR description :)

Reply:
@ondrejmirtes should the delta files all (except the original one where this was copied over) updated in that regard?

Reply:
I don't care, I never read that comment :) The filename says it all.